src/Raven.Server/Json/BlittableJsonTextWriterExtensions.cs

```diff
@@ -1302,6 +1302,10 @@ public static void WriteIndexDefinition(this AbstractBlittableJsonTextWriter wri
             else
                 writer.WriteNull();
             writer.WriteComma();
+            
+            writer.WritePropertyName(nameof(indexDefinition.ArchivedDataProcessingBehavior));
+            writer.WriteString(indexDefinition.ArchivedDataProcessingBehavior?.ToString());
```

Comment:
If `indexDefinition.ArchivedDataProcessingBehavior?` is null then you need to call `writer.WriteNull()` see above piece of code

Reply:
done